votor/src/voting_utils.rs

```diff
@@ -30,8 +30,10 @@ pub enum GenerateVoteTxResult {
     // hot spare validator, not eligble for refresh
     // until set identity is invoked
     HotSpare,
+    // Wait to vote slot is not reached
+    WaitToVoteSlot,
     // failed generation, eligible for refresh
-    Failed,
+    Failed(String),
```

Comment:
Should we call this `FailedToGenerate`? Because it looks like we call out some specific failures in the enum but then lump some into this `Failed` enum and then differentiate based on the `String`

Reply:
Related - would it make more sense to call out all the appropriate categories in the enum? Or possibly a sub-enum? And then we could attach the error strings to the enum, similar to what was done for `VoteError`